### PR TITLE
docs: Disable workloads Settting

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR explains how to disable the Palette agent from sending workloads to the control plane.

CleanShot 2023-05-17 at 16 27 05

## Review Changes

💻 [Preview URL](https://deploy-preview-1296--docs-spectrocloud.netlify.app/troubleshooting/nodes#paletteagentsworkloadpayloadsizeissue)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PE-1530)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1296#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md [hunk 1]
```diff
@@ -75,13 +75,17 @@ Palette 3.4.0 has various security upgrades, better support for multiple Kuberne
...
 ### Bug Fixes
 
 - A bug that caused issues with the deletion of a cluster's profile manifest has been successfully fixed. Manifests are now correctly deleted when removed from a cluster profile.
 
 
 - The problem with Palette not removing namespaces when removing a layer from a cluster profile has been resolved.
 
+
+
+- You can now configure the behavior of the Palette agent to disable sending workload reports to the Palette control plane. This addresses scenarios where large clusters with many nodes exceed the 1 MB payload threshold resulting in agent failures. Refer to the [Nodes Troubleshooting](/troubleshooting/nodes#paletteagentsworkloadpayloadsizeissue) for guidance on disabling the workload report feature.
```
Comment (ritawatson) [inline content/docs/00-release-notes.md:88]:
```suggestion
- You can now configure the behavior of the Palette agent to disable sending workload reports to the Palette control plane. This addresses scenarios where large clusters with many nodes exceed the 1 MB payload threshold, resulting in agent failures. Refer to the [Nodes Troubleshooting](/troubleshooting/nodes#paletteagentsworkloadpayloadsizeissue) for guidance on disabling the workload report feature.
```

File: content/docs/14-troubleshooting/06-nodes.md [hunk 2]
```diff
@@ -29,25 +29,49 @@ Logs are provided in Palette for traceability. However, these logs may be lost w
...
 ## Scenario -  vSphere Cluster and Stale ARP Table
-Sometimes certain vSphere clusters run into issues where non-VIP nodes are unable to contact the VIP node, primarly because non-VIP nodes' ARP entries became stale.
 
-To minimize this situation, vSphere clusters deployed from Palette now come equiped with a daemonset that cleans the ARP entry cache every 5 minutes. The cleaning process will force the nodes to re-request an ARP entry of the VIP node periodically. This is done automatically without any user action.
+Sometimes vSphere clusters encounter issues where nodes with an assigned Virtual IP Address (VIP) cannot contact the node with a VIP. The problem is caused by non-VIP nodes' Address Resolution Protocol (ARP) entries becoming stale.
 
-You can verify the cleaning process by running the following command on non-VIP nodes and observe that ARP cache is never older than 300 seconds:
+To minimize this situation, vSphere clusters deployed through Palette now have a daemon set that cleans the ARP entry cache every five minutes. The cleaning process will force the nodes to re-request an ARP entry of the VIP node periodically. This is done automatically without any user action.
```
Comment (ritawatson) [inline content/docs/14-troubleshooting/06-nodes.md:40]:
```suggestion
To minimize this situation, vSphere clusters deployed through Palette now have a daemon set that cleans the ARP entry cache every five minutes. The cleaning process forces the nodes to periodically re-request an ARP entry of the VIP node. This is done automatically without any user action.
```

File: content/docs/14-troubleshooting/06-nodes.md [hunk 3]
```diff
@@ -29,25 +29,49 @@ Logs are provided in Palette for traceability. However, these logs may be lost w
...
-
+<br />
 
 # Clusters
 
 ## Scenario -  vSphere Cluster and Stale ARP Table
-Sometimes certain vSphere clusters run into issues where non-VIP nodes are unable to contact the VIP node, primarly because non-VIP nodes' ARP entries became stale.
 
-To minimize this situation, vSphere clusters deployed from Palette now come equiped with a daemonset that cleans the ARP entry cache every 5 minutes. The cleaning process will force the nodes to re-request an ARP entry of the VIP node periodically. This is done automatically without any user action.
+Sometimes vSphere clusters encounter issues where nodes with an assigned Virtual IP Address (VIP) cannot contact the node with a VIP. The problem is caused by non-VIP nodes' Address Resolution Protocol (ARP) entries becoming stale.
```
Comment (ritawatson) [inline content/docs/14-troubleshooting/06-nodes.md:38]:
```suggestion
Sometimes vSphere clusters encounter issues where nodes with an assigned Virtual IP Address (VIP) cannot contact the node with a VIP. The problem is caused by Address Resolution Protocol (ARP) entries becoming stale on non-VIP nodes.
```

File: content/docs/14-troubleshooting/06-nodes.md [hunk 4]
```diff
@@ -29,25 +29,49 @@ Logs are provided in Palette for traceability. However, these logs may be lost w
...
+) for troubleshooting guidance. 
+
+<br />
 
-If your EKS cluster  worker pool ends up in `Failed` or `Create Failed` or `Error nodes failed to join` state, please refer to this [Amazon provided Runbook](https://docs.aws.amazon.com/systems-manager-automation-runbooks/latest/userguide/automation-awssupport-troubleshooteksworkernode.html
-)
+## Palette Agents Workload Payload Size Issue
+
+
+A cluster comprising many nodes can create a situation where the workload report data the agent sends to Palette exceeds the 1 MB threshold and fails to deliver the messages. If the agent encounters too many workload report deliveries, the agent container may transition into a  *CrashLoopBackOff* state.
```
Comment (ritawatson) [inline content/docs/14-troubleshooting/06-nodes.md:61]:
```suggestion
A cluster comprised of many nodes can create a situation where the workload report data the agent sends to Palette exceeds the 1 MB threshold and fails to deliver the messages. If the agent encounters too many workload report deliveries, the agent container may transition into a  *CrashLoopBackOff* state. 
```

**Review by ritawatson — APPROVED** [review]
LGTM with comments.

**Review by ritawatson — APPROVED** [review]
LGTM with comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1296#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1296#pullrequestreview-1431846661" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1296#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1296#pullrequestreview-1431846661" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**